src/storybook/html.stories.tsx

```diff
@@ -7,6 +7,41 @@ const storyMeta: Meta = {
   title: 'Demo/Html',
 }
 
+const Content: React.FC<{ text: string }> = ({ text }) => {
+  const [count, setCount] = React.useState(1)
+
+  React.useEffect(() => {
```

Comment:
in other stories we use `useState` and `useEffect` directly - without `React.`

Reply:
thx, missed it while prototyping